run--02a--run-and-validate-SAE.Rmd

```diff
@@ -413,40 +383,12 @@ if (rerun_sae) {
 }
 ```
 
-```{r save SAE ingredients}
```

Comment:
We actually need part of this chunk, saving if rerun_sae is TRUE and loading if rerun_sae is FALSE. Otherwise the next chunks would not run.